connectors/debezium-postgres-source/3.0.1.6/debezium-postgres-source.md

```diff
@@ -0,0 +1,157 @@
+| `database.history.pulsar.service.url` | false    | null    | Pulsar cluster service URL for history topic.                                                                                                                                                                                                                              |
+| `pulsar.service.url`                  | false    | null    | Pulsar cluster service URL.                                                                                                                                                                                                                                                |
+| `offset.storage.topic`                | false    | null    | Record the last committed offsets that the connector successfully completes.                                                                                                                                                                                               |
+
+## Advanced features
+
+### Converter options
+
+- org.apache.kafka.connect.json.JsonConverter
+
+  The`json-with-envelope` config is valid only for the JsonConverter. By default, the value is set to false. When the `json-with-envelope` value is set to false, the consumer uses the schema `Schema.KeyValue(Schema.AUTO_CONSUME(), Schema.AUTO_CONSUME(), KeyValueEncodingType.SEPARATED)`, and the message only consists of the payload.
+  When the `json-with-envelope` value is set to true, the consumer uses the schema `Schema.KeyValue(Schema.BYTES, Schema.BYTES`, and the message consists of the schema and the payload.
+
+- org.apache.pulsar.kafka.shade.io.confluent.connect.avro.AvroConverter
+
+  If you select the AvroConverter, the consumer uses the schema `Schema.KeyValue(Schema.AUTO_CONSUME(), Schema.AUTO_CONSUME(), KeyValueEncodingType.SEPARATED)`, and the message consists of the payload.
+
+
+### Used topic on Pulsar
+Currently, the destination topic (specified by the `destination-topic-name` option ) is a required configuration but it is not used for the Debezium connector to save data. The Debezium connector saves data on the following 4 types of topics:
+
+- One topic for storing the database metadata messages. It is named with the database server name ( `database.server.name`), like `public/default/database.server.name`.
+- One topic (`offset.storage.topic`) for storing the offset metadata messages. The connector saves the last successfully-committed offsets on this topic.
+- (Option) One topic (`database.history.pulsar.topic`) for storing the database history information. The connector writes and recovers DDL statements on this topic.
+- One per-table topic. The connector writes change events for all operations that occur in a table to a single Pulsar topic that is specific to that table.
```

Comment:
The format for this topic is for example like: `mydbserver.public.io-test` right? Maybe it's better to add the topic name format here.